### PR TITLE
Settings auto save

### PR DESCRIPTION
**Pull request by 1Spinne**
This automatically saves settings so you dont have to press "save"

### CODE REVIEW

File: src/app/desktop/windows/settings/settings.component.html [hunk 1]
```diff
@@ -3,33 +3,33 @@
...
       <label>
-        <input formControlName="terminalLsPrefix" type="checkbox">
+        <input (change)="saveSettings()" formControlName="terminalLsPrefix" type="checkbox">
       </label>
     </div>
     <div class="buttons-container">
       <div class="side-buttons row">
         <button (click)="resetSettings()">Reset Settings</button>
-        <button (click)="saveSettings()">Save</button>
+        <button>Save</button>
```
Comment (MarcelCoding) [inline src/app/desktop/windows/settings/settings.component.html:32]:
```suggestion
        <button title="This is fake :D All values are automatically saved.">Save</button>
```

File: src/app/desktop/windows/settings/settings.component.ts [hunk 2]
```diff
@@ -1,10 +1,10 @@
-import {Component, OnInit, Type} from '@angular/core';
```
Comment (MarcelCoding) [inline src/app/desktop/windows/settings/settings.component.ts:1]: You didn't change anything here, so don't commit it.

File: src/app/desktop/windows/settings/settings.component.html (hunk 1 above)
Comment (MarcelCoding) [reply src/app/desktop/windows/settings/settings.component.html:32]: Optional

**Review by MarcelCoding — CHANGES_REQUESTED** [review]

**Comment by 1Spinne** [comment]
I will delete this PR because just look at the files and it's not done. I have to change a few more things

**Pull request closed without merging**